App/Models/Task.cs

```diff
@@ -16,4 +16,20 @@ public Task(int id, string title, string description, DateTimeOffset created, Da
         Due = due;
         IsDone = isDone;
     }
+    
+    public override bool Equals(object obj)
+    {
+        return obj is Task task &&
+            Id == task.Id &&
+            Title == task.Title &&
+            Description == task.Description &&
+            Created.ToString() == task.Created.ToString() &&
+            Due.ToString() == task.Due.ToString() &&
+            IsDone == task.IsDone;
+    }
+
+    public override int GetHashCode()
+    {
+        throw new NotImplementedException();
```

Comment:
In case of overriding Equals method appropriate implementation of GetHashCode method should be provided. Let's discuss the reason on Tuesday